templates/certification_request/home.html.twig

```diff
@@ -1,54 +1,59 @@
+            {% if app.user.isCertified %}
+                <img src="{{ asset('/images/icons/certification/certified-lg.svg') }}" alt="certif-img"/>
+                <div class="title">Votre profil est certifié</div>
+
+            {% elseif app.user.certificationRequests.hasBlockedCertificationRequest %}
+                <img src="{{ asset('images/icons/certification/blocked.svg') }}" alt="certif-img"/>
+                <div class="title b__nudge--bottom">Demande de certification bloquée</div>
+                <p>Vous ne pouvez pas refaire de nouvelle demande pour l'instant.</p>
+
+            {% elseif app.user.certificationRequests.hasPendingCertificationRequest %}
+                <img src="{{ asset('/images/icons/certification/on-waiting.svg') }}" alt="certif-img"/>
+                <div class="title b__nudge--bottom">Demande de certification en attente</div>
+                <p>Votre demande est en attente d'approbation par nos équipes.</p>
+
+            {% elseif app.user.certificationRequests.hasRefusedCertificationRequest %}
+                {% set refusedCertificationRequest = app.user.certificationRequests.getRefusedCertificationRequests.first %}
+
+                <img src="{{ asset('/images/icons/certification/blocked.svg') }}" alt="certif-img"/>
+                <div class="title b__nudge--bottom">Demande de certification refusée</div>
+                <p>Votre demande de certification a été refusée.<br />
+                {% if refusedCertificationRequest.refusalReason %}
+                    <p>
+                        Raison:
+                        {% if refusedCertificationRequest.isRefusedWithOtherReason %}
+                            {{ refusedCertificationRequest.customRefusalReason }}
+                        {% else %}
+                            {{ refusedCertificationRequest.refusalReasonKey|trans }}
+                        {% endif %}
+                    </p>
+                {% endif %}
+                Vous pouvez <a class="link--blue--dark link--no-decor" href="{{ path('app_certification_request_form') }}">réessayer</a>.</p>
 
-            {% if app.user.birthdate %}
-                <a href="{{ path('app_certification_request_form') }}" class="btn btn--blue b__nudge--top">Certifier mon profil</a>
             {% else %}
-                <div class="b__nudge--top-40 alert alert--error">Malheureusement, nous ne connaissons pas votre date de naissance et nous en avons besoin pour certifier votre identité. Mettez à jour votre profil avant de continuer.</div>
-                <p><a href="{{ path('app_user_edit') }}" class="btn btn--blue b__nudge--top">METTRE À JOUR MON PROFIL</a></p>
+                <img src="{{ asset('/images/icons/certification/not-certified.svg') }}" alt="certif-img"/>
+                <div class="title b__nudge--bottom">Votre profil n'est pas encore certifié.</div>
+                <p>Certifiez le avec une pièce d'identité, ça ne prend qu'une minute.</p>
```

Comment:
```suggestion
                <p>Certifiez-le avec une pièce d'identité, ça ne prend qu'une minute.</p>
```

Reply:
hmm I'm not sure about that

Reply:
I'm sure 😄: http://www.visezjuste.uottawa.ca/pages/orthographe/pronoms_personnels.html

Reply:
haha ok, thanks !